test/test-str.lisp

```diff
@@ -172,7 +172,8 @@
 (test trim
   (is (string= "rst" (trim "  rst  ")))
   (is (string= nil (trim nil)))
-  (is (string= "rst" (trim "drste" :char-bag "de"))))
+  (is (string= "rst" (trim "drste" :char-bag "de")))
+  (is (string= "rst" (trim "　　rst　　　"))))
```

Comment:
This renders on Github as normal-looking spaces, but they aren't. Emacs renders them clearly with a cyan underline.